File: assignment20.py
list_of_nums = [1,2,3,4,5]

#this is assuming that the list is forever constant while incrememnting
def monotonicSequences(list_of_nums):
    if list_of_nums[0] > list_of_nums[-1]:
        print("the list of numbers is decreasing")
    elif list_of_nums[0] < list_of_nums[-1]:
        print("the list of numbers is increasing")

monotonicSequences(list_of_nums)

File: javascript_assignments/assignment20.js
let list_of_nums = [1,2,3,4,5]

function monotonicSequences(list_of_nums){

    //fix the if/then section tomorrow
    if (list_of_nums[0] > list_of_nums[-1]){
        console.log("the list of numbers is decreasing")
    } else if (list_of_nums[0] < list_of_nums[-1]){
        console.log("the list of numbers is increasing")
    }   
}

monotonicSequences(list_of_nums)